README.md

```diff
@@ -1,2 +1,27 @@
-# optee_examples
-OP-TEE Sample Applications
+# OP-TEE Sample Applications
```

Comment:
OK, indeed that can be used as an index to know were to look. I'd like to have no more than one sentence per example. For example:
- hello_world: prints some trace message and increments a parameter passed by value
- random: calls TEE_GenerateRandom() to fill a buffer passed by address

etc. ...you get the idea.